docs/REST API/rest-api.md

```diff
@@ -1717,6 +1719,7 @@ _Available in Fleet Premium._
 | macos_setup         | object  | See [`mdm.macos_setup`](#mdm-macos-setup). |
 | macos_settings         | object  | See [`mdm.macos_settings`](#mdm-macos-settings). |
 | windows_settings         | object  | See [`mdm.windows_settings`](#mdm-windows-settings). |
+| client_url         | string  | Update this URL if you're self-hosting Fleet and you want your hosts to talk to a different URL for MDM features (default: base URL of the Fleet instance provided during setup). |
```

Comment:
```suggestion
| client_url         | string  | Update this URL if you're self-hosting Fleet and you want your hosts to talk to a different URL for MDM features (default: base URL of the Fleet instance provided during setup). |
```
@noahtalerman does this mean it uses the base URL of the Fleet instance if empty, or that it gets filled in with the Fleet URL upon setup or migration & has to be manually updated if that base URL ever changes?

Reply:
IMO would be a better user experience to allow this to be empty and then default to the base URL if empty, rather than requiring the user to update in 2 places if they don't need a custom client URL and their base URL changes, which would look more like:
```suggestion
| client_url         | string  | Update this URL if you're self-hosting Fleet and you want your hosts to talk to a different URL for MDM features. (If not configured, `server_url` will be used instead.) |
```

Reply:
> better user experience to allow this to be empty and then default to the base URL if empty, rather than requiring the user to update in 2 places if they don't need a custom client URL and their base URL changes

@rachaelshaw agreed!

Maybe it's clearer w/o the "provided during setup?"

(If not configured, hosts will use the base URL of the Fleet instance.)